src/pages/registration/FileList.tsx

```diff
@@ -99,36 +95,56 @@ export const FileList = ({ title, files, uppy, remove, baseFieldName }: FileList
+                        maxWidth="xl"
                         modalTitle={t('registration.files_and_license.licenses')}
                         modalDataTestId={dataTestId.registrationWizard.files.licenseModal}
                         buttonDataTestId={dataTestId.registrationWizard.files.licenseHelpButton}>
-                        <Typography sx={{ mb: '1rem' }}>
-                          {t('registration.files_and_license.file_and_license_info')}
-                        </Typography>
-                        {activeLicenses.map((license) => (
-                          <Box key={license.id} sx={{ mb: '1rem', whiteSpace: 'pre-wrap' }}>
+                        <Trans
+                          i18nKey="registration.files_and_license.file_and_license_info"
+                          components={{
+                            p: <Typography gutterBottom />,
+                            ccLink: <OpenInNewLink href="https://creativecommons.org/licenses/" />,
+                          }}
+                        />
+                        <Divider sx={{ my: '1rem', borderColor: 'black', borderWidth: '1px' }} />
+                        <Box
+                          sx={{
+                            display: 'grid',
+                            gridTemplateColumns: { md: '1fr', lg: 'repeat(3, 1fr)' },
+                            gap: '1.5rem',
+                          }}>
+                          {activeLicenses.map((license) => (
+                            <Box key={license.id}>
+                              <Box sx={{ display: 'flex', alignItems: 'center', gap: '0.5rem', mb: '0.5rem' }}>
+                                <Box component="img" src={license.logo} alt="" sx={{ width: '5rem' }} />
+                                <Typography component="h2" variant="h3">
+                                  {license.name}
+                                </Typography>
+                              </Box>
+                              <Trans defaults={license.description} components={{ p: <Typography gutterBottom /> }} />
+                              {license.link && (
+                                <OpenInNewLink href={license.link}>
+                                  {t('licenses.read_more_about_license', { license: license.name })}
+                                </OpenInNewLink>
+                              )}
+                              {license.additionalInformation && (
+                                <Trans
+                                  defaults={license.additionalInformation}
+                                  components={{
+                                    p: <Typography sx={{ mt: '1rem' }} />,
+                                    link1: <OpenInNewLink href="https://lovdata.no/lov/2018-06-15-40/" />,
+                                  }}
+                                />
+                              )}
+                            </Box>
+                          ))}
+                          <Box>
```

Comment:
Om dette bare er for gruppering klarer vi oss uten MUI-avhengighet
```suggestion
                          <div>
```